readme.md

```diff
@@ -44,79 +47,54 @@ If you can verify that this solved your issue,
 try increasing this number until you find a happy middle ground. For reference, the default number
 for incremental builds is `256`, and for non-incremental builds `16`.
 
-You also cannot set `linker = "rust-lld.exe"`, as subsecond currently crashes when `linker` is set.
-
 </details>
 
-<details>
-<summary>
-macOS
-</summary>
+#### MacOS
+
 
 You're in luck! Everything should work out of the box if you use the default system linker.
 
-</details>
+
+#### Linux
+
+Prerequisites: `clang` and either `lld` or `mold`
 
 <details>
 <summary>
-Linux
+Minimal config
 </summary>
 
-Execute the following:
-```sh
-readlink -f "$(which cc)"
-```
-If this points to `clang`, you're good. Otherwise, we'll need to symlink it.
-Read the path returned by the following command:
-```sh
-which cc
-```
-and `cd` into it. For example,
-
-```sh
-$ which cc
-/usr/bin/cc
-$ cd /usr/bin
-```
-
-Assuming you have `clang` installed, run the following commands:
-
-```sh
-mv cc cc-real
-ln -s "$(which clang)" cc
-```
-Note that the above commands may require `sudo`.
-
-Now everything should work. If not, install `lld` on your system and add the following to your `~/.cargo/config.toml`:
-
+Create or edit `./.cargo/config.toml`, with this minimal config
 ```toml
 [target.x86_64-unknown-linux-gnu]
+linker = "clang"
 rustflags = [
-  "-Clink-arg=-fuse-ld=lld",
+  "-C",
+  "link-arg=-fuse-ld=lld",
 ]
 ```
 
-## Mold
+This repo includes `./.cargo/config_faster_builds.toml` which contains other compile-time improving configs known to work with subsecond.
 
-Subsecond works best with `lld` right now, so we recommend using that linker.
-If you want to try using `mold`, you can set it up like this:
+The prerequisites for that config are:
+- `nightly` rust: `rustup toolchain install nightly` 
+- `cranelift`, installed through `rustup component add rustc-codegen-cranelift-preview --toolchain nightly`
+- [mold](https://github.com/rui314/mold#installation) linker 
+- [sccache](https://github.com/mozilla/sccache#installation) build cache
```

Comment:
I'd prefer to have this as a comment in the `config.toml` file to keep this section slim